charts/dex.yaml

```diff
@@ -0,0 +1,56 @@
+image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
+
+nodeSelector:
+  node-role.kubernetes.io/infra: ''
+
+tolerations:
+  - key: "node-role.kubernetes.io/bootstrap"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/infra"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+replicas: 2
```

Comment:
Why? The default is 1, and the initial deployment is on Bootstrap only. Would make sense to handle this as we do for Prom/AM, i.e. use 1 replica for now, and make it configurable somehow (that'd be part of the #1996 epic I guess).

Reply:
Dex is stateless, so having 2 shouldn't harm. Even better, given the `updateStrategy`, there will always be one running even during upgrades and such.

Initially both may run on the `bootstrap` node, fine, but once there are more `infra` ones things can be spread.

So I agree with 2 :)

Reply:
OK, I get why having more than one can help for `updateStrategy`. Would still be useful to make it configurable in the long run, IMHO.